bazel/ray_deps_setup.bzl

```diff
@@ -319,3 +319,11 @@ def ray_deps_setup():
         # When you first run this tool, it'll recommend a sha256 hash to put here with a message like: "DEBUG: Rule 'hedron_compile_commands' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = ..."
         sha256 = "7fbbbc05c112c44e9b406612e6a7a7f4789a6918d7aacefef4c35c105286930c",
     )
+
+    auto_http_archive(
```

Comment:
Don't we need a sanity PR to build a wheel with this and run? (or is it supposed to be a follow up?)